.github/workflows/package-apisix-base-rpm-ubi.yml

```diff
@@ -0,0 +1,61 @@
+name: package apisix-base rpm for ubi
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - "v*"
+    paths-ignore:
+      - '*.md'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '*.md'
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Seems we don't need to schedule build. :)

Reply:
This same schedule trigger exists on other workflows as well. Should I remove them too?

Reply:
done